ci/unit_tests.bat

```diff
@@ -1,6 +1,47 @@
 @echo off
+setlocal enabledelayedexpansion
 
-setlocal
+if not exist %WORKSPACE% (
+  echo Error: env var WORKSPACE must be defined
```

Comment:
hmm...i ran into this error while test, but didn't get the echo to print in the cmd console.

Reply:
oh ok. do you mean testing in Jenkins? If so maybe `WORKSPACE` is not defined then? but it should be per the Jenkins build env var list page!?

Reply:
Sorry, i meant running locally I did not define `%WORKSPACE%` and had to review the code to understand the issue since the echo didn't print.  

I haven't tested in Jenkins yet.

Reply:
This seems to fix it.  Note - it works from cmd, but seems to not work from PS, Jenkins is not PS, and I _think_ .bat files are default executed by cmd.
```
if "%WORKSPACE%"=="" ( 
  echo Error: env var WORKSPACE must be defined
  exit /B 1
)
```

Reply:
fixed this.